terminus/geometry/latlon.py

```diff
@@ -64,55 +65,71 @@ def __repr__(self):
     def sum(self, other):
         return LatLon(self.lat + other.lat, self.lon + other.lon)
 
+    def curvature(self):
+        # Radius of planet curvature (meters)
+        curvature = Ecef.equatorial_radius() / math.sqrt(1 - Ecef.first_eccentricity_parameter() ** 2 * self._sin_lat() ** 2)
+        return curvature
+
+    def latlon_to_ecef(self):
+        x = self.curvature() * self._cos_lat() * self._cos_lon()
+        y = self.curvature() * self._cos_lat() * self._sin_lon()
+        z = ((Ecef.polar_radius() ** 2) / (Ecef.equatorial_radius() ** 2)) * self.curvature() * self._sin_lat()
+        return Ecef(x, y, z)
+
+    def global_to_ecef_rotation(self, enu):
```

Comment:
yes, move them to the Enu / Ecef class please and pass the latlon as a parameter